applications/asset_tracker_v2/sample.yaml

```diff
@@ -183,15 +183,15 @@ tests:
     integration_platforms:
       - nrf9160dk_nrf9160_ns
     platform_allow: nrf9160dk_nrf9160_ns nrf9161dk_nrf9161_ns
-    extra_args: SHIELD=nrf7002ek_nrf7002 OVERLAY_CONFIG=overlay-nrf7002ek-wifi-scan-only.conf
+    extra_args: SHIELD=nrf7002ek OVERLAY_CONFIG=overlay-nrf7002ek-wifi-scan-only.conf
```

Comment:
Asset Tracker documentation was not updated. Entire nrf should be searched for nrf7002ek_nrf7002 and fixed.